src/Practico_6/Ej_arreglos.java

```diff
@@ -0,0 +1,298 @@
+        for (int i = 0; i < arr.length; i++) {
+            System.out.println(arr[i]);
+        }
+    }
+
+    //Crear un método que muestre en pantalla todos los elementos de un array de float.
+    public static void imprimirArreglo(float[] arr){
+        for (int i = 0; i < arr.length; i++) {
+            System.out.println(arr[i]);
+        }
+    }
+
+    //Crear un método que reciba una arreglo, y retorne el promedio de los mismos.
+    public static double promedio(double[] arr){
+        double suma = 0;
+
+        for (int i = 0; i < arr.length; i++) {
+            suma += arr[i];
+        }
+        return suma / arr.length;
+    }
+
+    //Crear un método que reciba un arreglo de enteros, y retorne el menor.
+    private static int menor(int[] arr){
+        int min = Integer.MIN_VALUE;
+
+        for (int i = 0; i < arr.length; i++) {
+            if(min > arr[i]){
+                min = arr[i];
+            }
+        }
+        return min;
+    }
+
+    //Crear un método que reciba un arreglo de enteros, y retorne el mayor.
+    private static int mayor(int[] arr){
+        int max = Integer.MAX_VALUE;
+
+        for (int i = 0; i < arr.length; i++) {
+            if(max < arr[i]){
+                max = arr[i];
+            }
+        }
+        return max;
+    }
+    /*
+    Crear un método llamado hayVacantes que reciba un arreglo de booleanos y retorne verdadero si
+    alguna de sus posiciones esta en true, de lo contrario, deberá retornar false.
+    * */
+    private static boolean hayVacantes(boolean[] arr){
+        boolean hay = false;
+
+        for (int i = 0; i < arr.length; i++) {
+            if(arr[i] == true){
+                hay = true;
+            }
+        }
+        return hay;
+    }
+    /*
+    Crear un método reciba un arreglo de enteros por parámetro y que solicite el ingreso de un valor.
+    Debe retornar true si y sólo si el valor ingresado por el usuario se encuentra en el arreglo.
+    * */
+    private static boolean existeValor(int[] arr){
+        boolean existe = false;
+        Scanner input = new Scanner(System.in);
+        int valor = 0;
+
+        System.out.println("Ingrese el valor a buscar: ");
+        valor = input.nextInt();
+
+        for (int i = 0; i < arr.length; i++) {
+            if(valor == arr[i]){
+                existe = true;
+            }
+        }
+        return existe;
+    }
+
+    /*
+    Crear un método que reciba un arreglo de booleanos y retorne true si y sólo si,
+    todos los elementos son verdaderos.
+    * */
+    private static boolean todosVerdaderos(boolean[] arr){
+        boolean todos_verdaderos = false;
+        int c_verdaderos = 0;
+
+        for (int i = 0; i < arr.length; i++) {
+            if(arr[i] == true){
+                c_verdaderos++;
+            }
+        }
+
+        if(c_verdaderos == arr.length){
+            todos_verdaderos = true;
+        }
+        return todos_verdaderos;
```

Comment:
Podes hacerlo mas facil, un for, si encontras un false, retornas falso.. si sale del for, entonces son todos verdaderos, y retornas true

Reply:
Si es cierto jaja, ya lo modifiqué